specs.js

```diff
@@ -18,92 +40,93 @@ describe('when using trimperson api', () => {
     })
   })
 
-  Then('it should have created a trimperson object', () => expect(trim).to.be.a('object'))
-
-  describe('using the getContainer function', () => {
-    Then('the function should exist on the trim instance', () => expect(trim.getContainer).to.be.a('function'))
-
-    describe('when invoking the getContainer funciton', () => {
-      var getContainerMock
-      var fakeID = 'fakecontainerid'
-      var dataReturn
-      var errReturn
-
-      Given(() => {
-        getContainerMock = nock(mockURL)
-          .get('/GetContainer')
-          .query({trimid: fakeID, securityToken: mockToken})
-          .reply(200, {
-            containerNo: 'someid',
-            subContainers: []
-          })
-      })
+  describe('You should be able to call `createRecord` with 5 arguments (include alternativeContainers)', () => {
+    var title, containerId, extensionType, fileData, alternativeContainers, createRecordMock
 
-      When((done) => {
-        trim.getContainer(fakeID, (err, data) => {
-          dataReturn = data
-          errReturn = err
-          done()
-        })
-      })
+    Given(() => title = 'inside-out')
+    Given(() => containerId = 'asdfjk')
+    Given(() => extensionType = '.mp4')
+    Given(() => alternativeContainers = ['himark'])
+    Given(() => fileData = VALID_FILEDATA)
 
-      Then('it should resolve without error', () => {
-        expect(errReturn).not.to.exist
-        expect(dataReturn).to.exist
-        expect(dataReturn.containerNo).to.equal('someid')
-        expect(dataReturn.subContainers).to.be.a('array')
-        getContainerMock.done() // throws an error if no request was recorded
+    var errReturn
+    var dataReturn
+    Given(() => {
+      createRecordMock = nock(mockURL)
+          .filteringRequestBody(R.always('*'))
+          .post('/AddRecordToTrim', '*')
+          .query({securityToken: mockToken})
+          .reply(201, {RecordNo: VALID_RECORD_NO})
+    })
+    When((done) => {
+      trim.createRecord(title, containerId, extensionType, fileData, alternativeContainers, function (err, data) {
+        errReturn = err
+        dataReturn = data
+        done()
       })
     })
+    Then('it should be able to successfully use the function', (done) => {
```

Comment:
I needed a then to fit in with the whole describe -> given -> when -> then paradigm. I think so at least. I haven't used this particular paradigm before.